packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/transforms.js

```diff
@@ -244,6 +246,20 @@ describe('EDiscovery Transform Tests', () => {
         return result;
       });
 
+      it('Decrypt function throws an exception', () => {
+        ctx.webex.internal.encryption.decryptText = sinon.stub().returns(Promise.reject(new Error('@@@@@@')));
+
+        const result = Transforms.decryptReportContent(ctx, object, reportId)
+          .then(() => {
+            assert.fail('Decrypt did not fail as expected');
+          })
+          .catch(() => {
+            assert.isDefined(activity.error);
```

Comment:
Adding a check for an error scenario. Other scenarios such as `onBehalfOf` being undefined are already in place.